app/views/devise/mailer/password_change.html.erb

```diff
@@ -0,0 +1,3 @@
+<p>Hello <%= @resource.email %>!</p>
```

Comment:
I see an opportunity to dry this up, maybe let's create a "welcome" partial that takes a user as an argument?